src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String [] dateFromArray = dateFrom.split("\\.");
+        LocalDate dataFromFormatted = LocalDate.parse(dateFromArray[2]
+                + "-" + dateFromArray [1] + "-" + dateFromArray[0]);
+        String [] dateToArray = dateTo.split("\\.");
+        LocalDate dataToFormatted = LocalDate.parse(dateToArray[2]
+                + "-" + dateToArray [1] + "-" + (Integer.parseInt(dateToArray[0]) + 1));
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)